src/components/znkExercise/core/znkExerciseDrv.js

```diff
@@ -344,15 +344,13 @@
                              *  INIT
                              * */
                             scope.actions.setSlideDirection(scope.settings.initSlideDirection);
-                            if(scope.settings.initForceDoneBtnDisplay === null){
-                                if(scope.settings.viewMode === ZnkExerciseViewModeEnum.REVIEW.enum){
-                                    scope.actions.forceDoneBtnDisplay(false);
-                                }else{
-                                    scope.actions.forceDoneBtnDisplay(scope.settings.initForceDoneBtnDisplay);
-                                }
-                            }else{
+
+                            if(scope.settings.viewMode === ZnkExerciseViewModeEnum.REVIEW.enum){
```

Comment:
i think that review mode by definition is always after complete without done button, there's no point for done button if it's all complete. you think i should change it back?